src/networks.json

```diff
@@ -1880,5 +1880,23 @@
     },
     "start": 43317,
     "logo": "ipfs://QmXvum7SNVaAqAc2jNzR1NpNZN1GGvNaKWydg8a1GEDQ7y"
+  },
+  "9001": {
+    "key": "9001",
+    "name": "Evmos Mainnet",
+    "shortName": "EVMOS",
+    "chainId": 9001,
+    "network": "mainnet",
+    "multicall": "0x2764b262E60D3Ac96999F7e73b0628B09CbB367E",
```

Comment:
https://blockscout.evmos.org/address/0x2764b262E60D3Ac96999F7e73b0628B09CbB367E/contracts#address-tabs Can verify and publish code here?

Reply:
>  evmos informed me no public archival nodes will exist prior to the restart. POST restart, the above should do it.
What blockheight do you need?

@NoahMarconi It should atleast work from multicall contract creation block, i.e 59316 in this case